src/types/quizzes.d.ts

```diff
@@ -83,6 +83,10 @@ export interface QuizResultsResponse extends Record<string, any> {
   quiz_version_id: string;
   quiz_session_id: string;
   quiz_id: string;
+  quiz_selected_options: Array<{
+    value: string;
+    has_attribute: boolean
```

Comment:
Not important but does this need `;`?

Reply:
Good catch! baee98ce951c3aed48d4f0a294ed574d561c7186